src/ui/settings/ViewNotifications.qml

```diff
@@ -29,61 +29,100 @@ VPNFlickable {
+            showDivider: vpnFlickable.vpnIsOff
+            onClicked: {
+                if (vpnFlickable.vpnIsOff) {
+                    VPNSettings.captivePortalAlert = !VPNSettings.captivePortalAlert
+                }
+           }
+        }
+        VPNCheckBoxRow {
+            id: unsecuredNetworkAlert
+            objectName: "settingUnsecuredNetworkAlert"
+
+            width: parent.width
+            visible: VPNFeatureList.unsecuredNetworkNotificationSupported
+
+            //% "Unsecured network alert"
+            labelText: qsTrId("vpn.settings.unsecuredNetworkAlert")
+            //% "Get notified if you connect to an unsecured Wi-Fi network"
+            subLabelText: qsTrId("vpn.settings.unsecuredNetworkAlert.description")
 
-    VPNCheckBoxAlert {
-        anchors.top: VPNFeatureList.unsecuredNetworkNotificationSupported ? unsecuredNetworkAlert.bottom : captivePortalAlert.bottom
-        visible: !vpnFlickable.vpnIsOff
+            isChecked: (VPNSettings.unsecuredNetworkAlert)
+            isEnabled: vpnFlickable.vpnIsOff
+            showDivider: vpnFlickable.vpnIsOff
+            onClicked: {
+                if (vpnFlickable.vpnIsOff) {
+                    VPNSettings.unsecuredNetworkAlert = !VPNSettings.unsecuredNetworkAlert
+                }
+           }
+        }
+        VPNCheckBoxAlert {
+
+            visible: !vpnFlickable.vpnIsOff
+
+            //% "VPN must be off to edit these settings"
+            //: Associated to a group of settings that require the VPN to be disconnected to change
+            errorMessage: qsTrId("vpn.settings.vpnMustBeOff")
+        }
+
+        VPNCheckBoxRow {
+            id: switchServersAlert
+            objectName: "switchServersAlert"
+            visible: VPNFeatureList.notificationControlSupported
+
+            width: parent.width
+
+            //% "Server Switchting Notification"
+            labelText: qsTrId("vpn.settings.notification.serverSwitch")
+            //% "Get notified when you successfully switched Servers"
+            subLabelText: qsTrId("vpn.settings.notification.serverSwitch.description")
 
-        //% "VPN must be off to edit these settings"
-        //: Associated to a group of settings that require the VPN to be disconnected to change
-        errorMessage: qsTrId("vpn.settings.vpnMustBeOff")
+            isChecked: (VPNSettings.serverSwitchNotification)
+            showDivider: true
+            onClicked: {
+                VPNSettings.serverSwitchNotification = !VPNSettings.serverSwitchNotification
+           }
+        }
+        VPNCheckBoxRow {
+            id: connectionChangeAlert
+            objectName: "connectionChangeAlert"
+            visible: VPNFeatureList.notificationControlSupported
+
+            width: parent.width
+
+            //% "Connection Change Notification"
+            labelText: qsTrId("vpn.settings.notification.connectionChange")
+            //% "Get notified when the Connection Status Changed"
```

Comment:
The capitalization all over the place here. _Changed_ should not be capitalized, and AFAIK there's no feature called "Connection Status", so I'm not sure why that's capitalized either.